.landscaper/gardenlet/example/imports.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: imports.gardenlet.landscaper.gardener.cloud/v1alpha1
+kind: LandscaperGardenletImport
+runtimeCluster:
+  apiVersion: landscaper.gardener.cloud/v1alpha1
+  kind: Target
+  spec:
+    type: landscaper.gardener.cloud/kubernetes-cluster
+    config:
+      kubeconfig: |
+        apiVersion:...
+        # here goes the kubeconfig of the target cluster
+
+gardenCluster:
```

Comment:
Why does this look different than `runtimeCluster`, i.e., why doesn't it have `apiVersion` and `target`?

Reply:
this is only an example file. I ll adapt it!